packages/next/src/build/webpack/loaders/next-flight-loader/index.ts

```diff
@@ -34,10 +34,7 @@ export function getAssumedSourceType(
 
   if (sourceType === 'auto') {
     if (detectedClientEntryType === 'auto') {
-      if (
-        clientRefs.length === 0 ||
-        (clientRefs.length === 1 && clientRefs[0] === '')
```

Comment:
We don't have the `''` case anymore?

Reply:
oh yea that will handle it well. It's just parsing the SWC parsed client refs comment.